controller_manager/test/hwi_switch_test.cpp

```diff
@@ -151,19 +152,24 @@ class SwitchBot : public hardware_interface::RobotHW
 
         for(std::list<hardware_interface::ControllerInfo>::const_iterator it = start_list.begin(); it != start_list.end(); ++it)
         {
-            for (std::set<std::string>::const_iterator res_it = it->resources.begin(); res_it != it->resources.end(); ++res_it)
+            if (it->claimed_resources.size() != 1)
+            {
+                ROS_FATAL("We expect controllers to claim resoures from only one interface. This should never happen!");
```

Comment:
This will go away when controllers starting claiming resource from more than one interface (in the future)?